docs/guide/snaps.md

```diff
@@ -61,8 +61,8 @@ Open up a new terminal window, and do the following:
   - This will install and/or use the recommended NVM version from `.nvmrc`.
 - Run `yarn install && yarn build:clean && yarn serve`.
   1. This first runs the commands needed to install dependencies.
-  1. If there is a `dist/` folder, it is then cleaned before the build command is run.
```

Comment:
We don't need to enumerate these for them to show correctly, so I believe it is done intentionally to simplify potentially changing the list in the future.

Reply:
OK resolved in latest commit.